GoMeet/src/main/webapp/test/SpecRunner.html

```diff
@@ -50,6 +50,7 @@
       <p>Votes table</p>
   </div> 
   <div id="map"></div></br>
+  <ul style="display: none;" id="popular-locations-container"></ul>
```

Comment:
Nit: It's not ideal to be putting in-line styling in HTML (https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns). But seeing as though it's already being done elsewhere in the file, I guess it's okay to leave it.

Reply:
Ahh, I see. I'll avoid it in the future :).